README.md

```diff
@@ -21,8 +21,12 @@ performance optimizations we’ve added along the way. Among those is a standard
 framerate-based renderer, a renderer for high-performance scrollable
 regions which works alongside the main renderer, and mouse support.
 
-To get started, see the tutorial below, the [examples][examples], the
-[docs][docs] and some common [resources](#libraries-we-use-with-bubble-tea).
+## Getting Started
+
+We recommend starting with our [basics tutorial](https://github.com/charmbracelet/bubbletea/tree/master/tutorials/basics) followed by our [commands tutorial](https://github.com/charmbracelet/bubbletea/tree/master/tutorials/commands) to understand what you can do with `tea.Cmd` and how they're used in BubbleTea. 
```

Comment:
It's also probably a bit too early to introduce the notion of a `tea.Cmd` to brand new users, so I'd drop the end of the sentence reading “to understand what you can do with `tea.Cmd`…” Additionally, I'd change possessive articles (our) to definite articles (the) to soften the sense of "us versus them" here.